app/src/main/java/com/example/coopertest/DatePreference.java

```diff
@@ -0,0 +1,98 @@
+package com.example.coopertest;
```

Comment:
Why are these 3 files in Java? Why do we need them? Couldn't this be imported from somewhere?

Reply:
Nop, I can import, so we have to have these file if we want the pop-up for date.

Reply:
You can't? I'm sure there must be something for that, there's no way every project that has calendar needs to have these files. At the very least they should be moved to a separate utils package so that we don't see them, but there must be a library for something - if not then I'd say it's better not to use the calendar at all and use some other input suitable for date

Reply:
I spend more than 6h to find a solution. Try and let's see. I think the best thing is to let like this. It's not good to spend time to redone something done. This project is realized in 2 weeks, so the objective is to put the maximum function in two weeks. You should change after the presentation. As you can see in class, Radu look often this kind of details

Reply:
I just realised something, sorry its so late...
We should not ask for specific date of birth at all - because we don't need it and it's private information, nobody wants to give it to a running app. 
Just the year of birth is completely enough, we don't need to be more accurate, the test levels are arbitrary anyway.
And it will make the code much much much simpler, we don't need all the calculations and 3rd party java files. Just something like a simple spinner with default of 2000 for year of birth. Or even a numeric keyboard will do.

What do you think?